src/hooks/useToggle.test.ts

```diff
@@ -0,0 +1,44 @@
+import {act, renderHook} from '@testing-library/react-hooks';
+import {useToggle} from '.';
+
+const setup = (initialValue: boolean) => {
```

Comment:
`setup` doesn't doesn't describe what this value is.  Something like `useToggleHook` would make more sense.  Then when you use it in a test it would read better as `useToggleHook(true)`.

Reply:
A `setup` function is a somewhat common testing convention, but I suppose in this case it could have a more descriptive name since it just does one thing

Reply:
Yeah I agree.  If we were doing more set up things and not just returning one hook `setup` would make more since but in this scenario we are just rendering the hook.

Reply:
I should mention, the idea of the `setup` function convention is that it's 'setting up' the initial 'state' for the test in a reusable way and it typically returns the values/utilities from whatever library is being used; in this case it's `renderHook`, if this was `testing-library/react` is would be `render`. 

So when I see a `setup` function in a test I know that it's essentially a shortcut

Reply:
When testing hooks we are testing them in isolation. If we are in the useDialog test file, we know we are rendering the useDialog hook. In the useToggle test file, the useToggle hook. 

If we scroll down and loose sight of what `setup` does, how can we maintain a reasonable mental model?
Generally it's a declarative name. 

We can safely assuming useToggle will be the hook being rendered.

What about `setupRenderHook`?

In other tests where setup does more than a single function maybe it becomes `setupRenderHookWithX`.

I like keeping `setup` since it's a general convention. I'd also like the `setup` function to give a little clue as to what it's setting up.

Reply:
Yeah that makes sense. I think `setupRenderHook` would work and if the setup is the same across enough test files we can export a general version from a `test-utils` file